lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -57,23 +55,24 @@ def test_get_authenticated_user_not_enrolled(self):
     def test_get_unauthenticated_user(self):
         self.client.logout()
         response = self.client.get(self.url)
-        assert response.status_code == 403
+        assert response.status_code == 401
 
     def test_get_unknown_course(self):
         url = reverse('course-home-progress-tab', args=['course-v1:unknown+course+2T2020'])
         response = self.client.get(url)
         assert response.status_code == 404
 
-    def test_masquerade(self):
-        user = UserFactory()
-        set_user_preference(user, 'time_zone', 'Asia/Tokyo')
-        CourseEnrollment.enroll(user, self.course.id)
-
-        self.switch_to_staff()  # needed for masquerade
-
-        # Sanity check on our normal user
-        assert self.client.get(self.url).data['user_timezone'] is None
-
-        # Now switch users and confirm we get a different result
-        self.update_masquerade(username=user.username)
-        assert self.client.get(self.url).data['user_timezone'] == 'Asia/Tokyo'
+    # TODO: (AA-212) implement masquerade
+    # def test_masquerade(self):
```

Comment:
the test has to be commented out? Why is that? It stopped working?

Reply:
It stopped working because I removed `user_timezone` from the API, so I commented it out because I'm about to pickup the masquerade ticket.

Reply:
Ah okay, yeah, as long as we reimplement before launch 👍